### ISSUE TITLE
Bad defaults (major bikeshedding alert!)

### ISSUE THREAD

**Issue by jrcii** [irssi/irssi#1237]
Hi guys,

I know this is an open source project so first and foremost: THANK YOU to everyone who spends and has spent time on this project. It is by far my favorite IRC client. Because I love it so much, I want it to be perfect. Due to a hard drive crash I lost my prior configuration that I slowly perfected over months. The thought of doing that over is painful. Maybe you'll disagree, but I'm just throwing this out there.

**My Idea of Sane Defaults**

- When you join a channel show only the number of nicks, not the entire list
- Each nick should get a unique color from a muted palette, yours gets a bright color, maybe even apply this to the nick and message so it's easy to separate stacked messages visually
- Nicks should be formatted like this:&nbsp;&nbsp;&nbsp; **nick:** message &nbsp;&nbsp;&nbsp; not this `<nick> message` What is this, XML?
- Message timestamps should be in 12 hour time (are we in the military or a robot? maybe European?)
- The default status bar format is horrendous. 
My current (default) status bar:

`[20:42] [jrc(+Zi) [1:Freenode/##aple(+Ccnrt)] [Act: 1]`

Why do you have to learn a secret code language to use this IRC client? I get that these are status flags for the channel and user but I'm a person not a parser. Even having used IRC for decades these flags don't come up enough to have them memorized.

How about this?

`8:44 PM jrc@freenode  1:Console 2:##programming`

Click the channel/window name to switch. If a channel has an update, the channel name becomes bold. If you have an unusual status, then you show it in plain english:

`8:44 PM jrc@freenode 1:Console 2:##programming (no voice)`

- Hide join/part spam unless the nick is someone who's spoken recently or mentioned your nick or you've mentioned their nick, this is especially crazy on big channels

I feel like the current defaults are probably favored by Unix wizards from the 70s or something. No defaults are going to be liked by everyone, and that's why we have configuration options, but I think these would be more popular and friendly to new users than what we have now.

**Comment by ailin-nemui** [comment]
make sure to have a backup of your config next time :-)

**Comment by dequis** [comment]
Yeah, nah, I disagree on all points except one.

**Comment by LemonBoy** [comment]
> Nicks should be formatted like this:    nick: message     not this <nick> message What is this, XML?

Don't be silly, that'd be malformed XML without a closing tag.

> Message timestamps should be in 12 hour time (are we in the military or a robot? maybe European?)

This is the best suggestion I've ever read, thanks for the laugh.

**Comment by Geolykt** [comment]
AM/PM is the most stupid thing since only few know what they mean past having to add 12 hours on top if it is PM. Also complicates other things honestly, the 24 hour standard should also be the default since it results in less characters being printed and as such less clutter

**Comment by jrcii** [comment]
> AM/PM is the most stupid thing since only few know what they mean past having to add 12 hours on top if it is PM. Also complicates other things honestly, the 24 hour standard should also be the default since it results in less characters being printed and as such less clutter

What makes sense just depends on where you are: https://i.imgur.com/BMs8vcZ.png If we wanted an objective solution maybe we could look at the geographic demographics of irssi users and correlate it with this map. Again I'm just bike shedding here so no big deal wither way.
